tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -122,14 +122,17 @@ def test_get_metric_extraction_config_single_alert(default_project: Project) ->
         config = get_metric_extraction_config(default_project)
 
         assert config
-        assert len(config["metrics"]) == 1
```

Comment:
Some changes in this file are code-refactoring to make future spec version changes a little easier.

For instance, do not check the length of the metrics if we assert the contents of `config["metrics"]` right after.

I'm going to open another PR that will include these changes to make this PR easier to review.
You can ignore changes in this module until I get the PR in place.